test/forms/LocationOfGoodsAuthorisationNumberSpec.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import forms.Constants.maxLocationOfGoodsAuthorisationNumberLength
+import forms.behaviours.StringFieldBehaviours
+import org.scalacheck.Gen
+import play.api.data.{Field, FormError}
+import models.domain.StringFieldRegex.alphaNumericRegex
+
+class LocationOfGoodsAuthorisationNumberSpec extends StringFieldBehaviours {
```

Comment:
Very minor but I guess this is technically `LocationOfGoodsAuthorisationNumberFormProviderSpec`

Reply:
Updated.